pavlo976/BuildMinder-App
# Feature/libraries

### What changed?
All needed libraries was installed to application
### How to test it?
Import functions from installed libraries and check the console on the website if they work properly
### Additional information
React-leaflet-cluster library doesn't work properly. When the component re-render, popup of marker flashes or dissapears completely
